port_ocean/utils/async_iterators.py

```diff
@@ -0,0 +1,41 @@
+import typing
+
+import aiostream
+
+
+async def stream_async_iterators_tasks(
```

Comment:
Maybe this should be
Merge async iterators?

Reply:
the main thing that it does is streaming the results from the iterators tasks. 
I feel like `merge` would just combine the tasks and return them, while stream actually performs the iterations and returns it as an async iterator